lib/ansible/module_utils/vmware.py

```diff
@@ -244,7 +244,7 @@ def gather_vm_facts(content, vm):
             'label': entry.deviceInfo.label,
             'macaddress': entry.macAddress,
             'ipaddresses': net_dict.get(entry.macAddress, None),
-            'macaddress_dash': entry.macAddress.replace(':', '-'),
+            'macaddress_dash': entry.macAddress.replace(':', '-') if entry.macAddress else None,
```

Comment:
this ternary operator is not compatible with python 2.4 and we should supply a such compat for Ansible 2.3 (not 2.4).
Can you fix it ?